@utahdts/utah-design-system/react/components/buttons/ConfirmationButton.jsx

```diff
@@ -58,29 +87,95 @@ function ConfirmationButton({
   promptChildren,
   size,
   type,
+  ...rest
 }) {
   const [isClicked, setIsClicked] = useState(false);
-  const buttonClick = () => {
-    setIsClicked(!isClicked);
-    onClick();
-  };
+  const timer = useRef(NaN);
```

Comment:
Lets discuss the Timer. What effect will this have on accessibility? It was a very unexpected behavior for me.

Reply:
One idea I just had was to start the timer after the focus leaves the button?

Reply:
Testing the button in screen reader worked well!

Reply:
✅  Kill the timer! 😢 🪦